app/build.gradle

```diff
@@ -150,7 +150,7 @@ dependencies {
     implementation 'androidx.multidex:multidex:2.0.1'
     implementation "androidx.appcompat:appcompat:1.1.0"
     implementation "androidx.cardview:cardview:1.0.0"
-    implementation "androidx.recyclerview:recyclerview:1.1.0-beta05"
+    implementation "androidx.recyclerview:recyclerview:1.1.0-rc01"
```

Comment:
Why?

Reply:
I didn't change it explicitly, maybe my android studio did it.
Shall I change it back @iamareebjamal ?

Reply:
> maybe my android studio did it.

Magic

Reply:
revert it